pkg/operator/certrotation/target.go

```diff
@@ -24,10 +24,11 @@ import (
 // TargetRotation rotates a key and cert signed by a CA. It creates a new one when <RefreshPercentage>
 // of the lifetime of the old cert has passed, or if the common name of the CA changes.
 type TargetRotation struct {
-	Namespace string
-	Name      string
-	Validity  time.Duration
-	Refresh   time.Duration
+	Namespace              string
+	Name                   string
+	Validity               time.Duration
+	Refresh                time.Duration
+	RefreshOnlyWhenExpired bool
```

Comment:
add a go doc please. `// normally rotation happens after 80% of validity. This allows to wait until expiration`.